samples/tab-calendar/ts/README.md

```diff
@@ -0,0 +1,161 @@
+---
+page_type: sample
+description: This sample shows tab calendar in Outlook apps.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- typescript
+extensions:
+ contentType: samples
+ createdDate: "20/06/2023 01:38:27 PM"
+urlFragment: officedev-microsoft-teams-samples-tab-calendar-ts
+---
+
+# Tab Calendar in Outlook
```

Comment:
Interact with Calendar in Outlook from Personal Tab

Reply:
Fixed